src/Perlang.Tests.Architecture/Perlang.Tests.Architecture.csproj

```diff
@@ -1,9 +1,5 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
-    <PropertyGroup>
-        <TargetFramework>net5.0</TargetFramework>
-    </PropertyGroup>
-
```

Comment:
(This project had a `TargetFramework` of its own for some reason. Doesn't make much sense, so might as well be removed.)